nature/styleguide/accessibility.md

```diff
@@ -3,7 +3,7 @@
 ## What is accessibility?
 Accessibility is the practice of designing products and services to be usable by all people. This includes people with vision, hearing, cognitive or motor impairments. Designing for accessibility also creates added benefits for all users. For example, high contrast, readable text makes it easier to read on a phone in bright sunlight.
 
-Ensuring our websites are accessible is the responsibility of everyone at Springer Nature. It is also required by law in many countries where we do business. As a research publisher, we have a duty to make human knowledge accessible for people with all abilities.
+Ensuring our websites are accessible is the responsibility of everyone at Springer Nature. It is also required by law in many countries where we do business. As a research publisher, we have a duty to make human knowledge accessible for people with all abilities. Accessibility is required to effectively opening the doors to discovery for all.
```

Comment:
I like this, though grammatically this should say "Accessibility is required to effectively **open** the doors to discovery for all"